idnits

```diff
@@ -529,6 +529,15 @@ BEGIN {
   # This applies to non-IETF streams
   bp["trust-28-dec-2009_6_b_ii_p3"] = "This document is subject to BCP 78 and the IETF Trust's Legal Provisions " "Relating to IETF Documents \\(https?://trustee\\.ietf\\.org/license-inf(o|o/)\\) " "in effect on the date of publication of this document\\.  " "Please review these documents carefully, as they describe your " "rights and restrictions with respect to this document\\.$"
   # ----------------------------------------------------------------------------------------------
+  bp["tlp6_copyright"] = "Copyright \\(c\\) \\(?[0-9][0-9][0-9][0-9]\\)?\\..+ IETF Intellectual Property " "Management Corporation and the persons identified as the document authors."
+	bp["tlp6_copyright_now"] = sprintf("Copyright \\(c\\) %s IETF Intellectual Property " "Management Corporation and the persons identified as the document authors.", year)
+  bp["tlp6_bcp_conformance"] =  bp["trust-28-dec-2009_6_a"]
+	bp["tlp6_ietf_tlp_section"] = "This document is subject to BCP 78 and the IETF IPM Technical Legal Provisions " "Relating to IETF Documents \\(https://www\\.ietf-ipm\\.org/license-inf(o|o/)\\) " "in effect on the date of publication of this document\\.  " "Please review these documents carefully, as they describe your " "rights and restrictions with respect to this document\\.  " "Code Components extracted from this document must include " "Revised BSD License text as described in Section 4.e of the " "Technical Legal Provisions and are provided without warranty as described " "in the Revised BSD License\\.$"
```

Comment:
Current xml2rfc PR has "Section 4.e of the IETF IPM Technical Legal Provisions and are". This is missing IETF IPM.